### PR TITLE
Add Bug Report

### PR DESCRIPTION
**Pull request by Nerox9**
Operating clone of bug report form UI. It is placed into the settings.

Github needs a configuration file which reads from environment variable `GITHUB_CONFIG` or by default `/home/ghaf/.config/ctrl-panel/config.toml`.

### CODE REVIEW

**Review by dmitry-erin — CHANGES_REQUESTED** [review]
UI changes required:
1) align left,
2) make the same indent,
3) add spacing between labels and entries.
Screenshot from 2024-12-09 09-46-55
Also run `cargo fmt ` to format all the code

File: src/styles/style.css [hunk 1]
```diff
@@ -73,6 +83,14 @@ label.title {
   color: #FFFFFF;
 }
 
+checkbutton>label {
+  color: #FFFFFF;
+}
+
+button.submit-button {
```
Comment (dmitry-erin) [inline src/styles/style.css:90]: not needed, the settings-button exists

File: src/ui/report.ui [hunk 2]
```diff
@@ -0,0 +1,721 @@
...
+                    <property name="height-request">30</property>
+                    <property name="label">Submit</property>
+                    <property name="valign">end</property>
+                    <property name="width-request">60</property>
+                    <signal name="clicked" handler="on_submit" swapped="yes"/>
+                    <style>
+                      <class name="settings-button"/>
+                    </style>
+                    <style>
+                      <class name="submit-button"/>
```
Comment (dmitry-erin) [inline src/ui/report.ui:707]: please, remove settings-button style is enough

File: src/ui/report.ui [hunk 3]
```diff
@@ -0,0 +1,721 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <template class="Report" parent="GtkBox">
```
Comment (dmitry-erin) [inline src/ui/report.ui:4]: There should be title label Bug report or similar like on other setting pages

File: src/ui/report.ui [hunk 4]
```diff
@@ -0,0 +1,721 @@
...
+                    </style>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="GtkBox">
+                <property name="halign">end</property>
+                <child>
+                  <object class="GtkButton" id="submit">
```
Comment (dmitry-erin) [inline src/ui/report.ui:697]: There should be space between the button and box

**Review by dmitry-erin — CHANGES_REQUESTED** [review]

File: src/styles/style.css (hunk 1 above)
Comment (Nerox9) [reply src/styles/style.css]: Style is replaced

File: src/ui/report.ui (hunk 3 above)
Comment (Nerox9) [reply src/ui/report.ui]: Title is added

File: src/ui/report.ui (hunk 4 above)
Comment (Nerox9) [reply src/ui/report.ui]: The parent box is removed

**Pull request merged**